### PR TITLE
Users guide review

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

File: content/users-guide/device-management-bundle/device-details.md [hunk 1]
```diff
@@ -147,6 +147,12 @@ Details on these additional menu items are provided where required.
 
 The **Alarms** tab provides information on the alarms of a device. See [Working with alarms](#alarm-monitoring) for detailed information on alarms.
 
+<a name="availability"></a>
+### Availability
+
+In addition to connection monitoring, {{< product-c8y-iot >}} offers a separate availability monitoring for machines, see [Availability](#monitoring-availability) for more information.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/device-details.md:153]: I would better rephrase the sentence a bit.

The **Availability** tab offers availability monitoring for machines, see [Monitoring and controlling devices > Availability](#monitoring-availability) for more information.

File: content/users-guide/enterprise-tenant-bundle/support-user-access.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ After a user has activated support access, the menu item changes to **Disable su
 
 > **Info:** If a user with tenant management admin permissions disables the support request, *all* support requests for the tenant will be disabled.
 
-The duration of the active support request can be globally configured in the {{< management-tenant >}} (default is 24 hours), see [Administration > Changing setting > Configuration settings](/users-guide/administration/#config-platform).
+The duration of the active support request can be globally configured in the {{< management-tenant >}} (default is 24 hours), see [Administration > Changing setting > Configuration settings](/users-guide/administration/#configuration-settings).
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/support-user-access.md:36]: config-platform is correct but the link text is wrong. Should be "Administration > Platform configuration settings"

File: content/users-guide/device-management-bundle/device-details.md (hunk 1 above)
Comment (JoshuaAdemola) [reply content/users-guide/device-management-bundle/device-details.md:153]: Fixed

File: content/users-guide/enterprise-tenant-bundle/support-user-access.md (hunk 2 above)
Comment (JoshuaAdemola) [reply content/users-guide/enterprise-tenant-bundle/support-user-access.md:36]: Fixed

File: content/users-guide/device-management-bundle/device-details.md (hunk 1 above)
Comment (JoshuaAdemola) [reply content/users-guide/device-management-bundle/device-details.md:153]: Fixed

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@JoshuaAdemola The English version of the admin-applications-properties screenshot is also "cut off" :-). Could you replace it here as well please?

**Comment by JoshuaAdemola** [comment]
@BeateRixen I replace the English version of the admin-application-properties screenshot in PR #268

**Review by MWindel — APPROVED** [review]

**Pull request merged**